src/user/user.service.ts

```diff
@@ -53,4 +53,10 @@ export class UserService {
     const user = await this.userRepository.update(updateUserPayload, userId);
     return user;
   }
+
+  async userIsVolunteer(id: number) {
+    const user = await this.userRepository.getById(id);
+    if (user?.role === 'volunteer') return true;
+    else return false;
```

Comment:
```suggestion
    if (user?.role === 'volunteer') return true;
    return false;
```
Let's get rid of useless else operator